webserver_test.go

```diff
@@ -0,0 +1,20 @@
+package links_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"links"
+)
+
+func TestCheckWebServer(t *testing.T) {
+	t.Parallel()
+	request, _ := http.NewRequest(http.MethodGet, "/check?site=https://bitfieldconsulting.com&no-recursion=true", nil)
```

Comment:
We don't ignore errors, even in tests (especially not in tests). And just `http.Get` would work here, wouldn't it?